src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/Histogram.jsx

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
 import {
-  BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip,
+  BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Label,
 } from 'recharts';
 import PropTypes from 'prop-types';
 
-// TODO - add legend, auto-scaling, sorting, etc
+// TODO - improve tickGap - e.g. the minTickGap={50} below needs to be dynamically calculated
 const Histogram = ({
   data,
   xAxisDataKey,
   barDataKey,
   chartWidth,
   chartHeight,
   barColor,
+  xAxisLegend,
 }) => (
   <BarChart
     width={chartWidth}
     height={chartHeight}
     data={data}
+    margin={{ top: 20, bottom: 65, right: 60 }}
   >
-    <XAxis dataKey={xAxisDataKey} />
+    <XAxis dataKey={xAxisDataKey} minTickGap={50} tickFormatter={(tick) => Math.round(tick * 10) / 10}>
```

Comment:
It looks like for some data, it renders an empty bar and NaN. Consider adding a check so if something is showing as NaN a more friendly user message renders? Like, 
```Sorry, unable to render chart with this selection``` or something. 
<img width="879" alt="image" src="https://user-images.githubusercontent.com/113449836/208769453-e9e340b1-f2e3-41e0-ad10-2cea25d0f09c.png">

Reply:
The error 
`Warning: Failed prop type: The prop selectedStudyPopulationCohort is marked as required in ContinuousCovariates, but its value is undefined. ` is solved.

Reply:
I also added a message for when there is no data:

![Screenshot 2022-12-21 at 15 39 15](https://user-images.githubusercontent.com/2900303/208930857-206ecf92-e6e7-4a39-957f-5ea0f970158c.png)